addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/receipt_header/receipt_header.xml

```diff
@@ -22,7 +22,7 @@
                     <div t-esc="props.data.cashier" />
                 </div>
                 <div t-if="props.data.trackingNumber and !props.data.bigTrackingNumber">
-                    <span class="fs-2" t-esc="props.data.trackingNumber" />
```

Comment:
why not just replace with `fs-1`?

Reply:
Changed the class to fs-1

Reply:
@vlst-odoo 
Referance Image after fix: 
![image](https://github.com/user-attachments/assets/942bd27d-ad3a-40af-ab87-c21ffdc4a903)

Reply:
> why not just replace with `fs-1`?

Looks like "tracking-number" is needed in other module for reference.